### PR TITLE
More minor typo fixes

### PR DESCRIPTION
**Pull request by tomioe**
## Description

Corrects a few minor typos around the project.
This is another "low-hanging fruit" commit, that aims to improve the overall quality. I am not (yet) able to contribute to code changes or improve/add any meaningful features but hopefully this small commit is still accepted.

## Contributing requirements

* [x] I read the contribution guidelines in [CONTRIBUTING.md](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/CONTRIBUTING.md).
* [x] I understand the BlinkenPerBit metric, and maximized it in this PR.
* [x] I selected `main` as the target branch.
* [x] All code herein is subjected to the license terms in [COPYING.txt](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/COPYING.txt).

### CODE REVIEW

File: include/globals.h [hunk 1]
```diff
@@ -57,7 +57,7 @@
 //              Apr-23-2021  v015       Davepl      Fixed fan effects
 //              Apr-23-2021  v016       Davepl      Fix max power limit!
 //              Apr-24-2021  v017       Davepl      Fix compressed frames - stable!
-//              May-01-2021  v018       Davepl      Put receive timeout back in, cRec'd to 0
+//              May-01-2021  v018       Davepl      Put recive timeout back in, cRec'd to 0
```
Comment (rbergen) [inline include/globals.h:60]: I believe "receive" **is** the correct spelling.

File: include/ledbuffer.h [hunk 2]
```diff
@@ -25,7 +25,7 @@
 //
 //   Provides a timestamped buffer of colordata.  The LEDBufferManager keeps
 //   N of these buffers in a circular queue, and each has a timestamp on it
-//   indicating when it becomes valid.
+//   indicating wheb it becomes valid.
```
Comment (rbergen) [inline include/ledbuffer.h:28]: I believe "when* is the correct spelling.

File: include/ledbuffer.h [hunk 3]
```diff
@@ -314,7 +314,7 @@ class LEDBufferManager
 
     // PeekOldestBuffer
     //
-    // Take a "peek" at the newest buffer, or nullptr if empty
+    // Take a "peek" at the newest buffer, or nullptr if emptyu
```
Comment (rbergen) [inline include/ledbuffer.h:317]: I believe "empty" is the correct spelling.

File: include/ledmatrixgfx.h [hunk 4]
```diff
@@ -60,7 +60,7 @@
 
 class LEDMatrixGFX : public Adafruit_GFX
 {
-  friend class LEDStripEffect;                                          // I might a shower after this lifetime first, but it needs access to the pixels to do its job, so BUGBUG expose this more nicely
+  friend class LEDStripEffect;                                          // I might a shower after this lifetime first, but it needs acess to the pixels to do its job, so BUGBUG expose this more nicely
```
Comment (rbergen) [inline include/ledmatrixgfx.h:63]: I believe "access" is the correct spelling.

File: include/spiffswebserver.h [hunk 5]
```diff
@@ -29,7 +29,7 @@
 //   adjusting effect paramters.  I'm in no way attached to it and it
 //   should likely be redone!
 //
-//   Server also exposes basic RESTful API for querying variables etc.
+//   Server also exposes basic RESTful API for querying variablesl etc.
```
Comment (rbergen) [inline include/spiffswebserver.h:32]: I believe "variables" is the correct spelling.

**Review by rbergen — COMMENTED** [review]
This PR seems to introduce a number of spelling regressions. Did you maybe apply your changes to a branch created from an older commit?

**Comment by tomioe** [comment]
Oh, absolutely... Had a bit of hiccup in terms of the branches, good catch on these.

**Comment by tomioe** [comment]
> This PR seems to introduce a number of spelling regressions. Did you maybe apply your changes to a branch created from an older commit?

Oh, absolutely... Had a bit of hiccup in terms of the branches, good catch on these.

**Pull request merged**